.github/workflows/lint.yaml

```diff
@@ -0,0 +1,39 @@
+name: Linting
```

Comment:
Since we are modifying the name, can we make all the extensions match? The other workflow has `yml`

Reply:
Np, done